src/patternfly/components/Page/page.scss

```diff
@@ -339,12 +339,10 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
     }
   }
 
-  &.pf-m-fill,
-  &:has(.#{$page}__main-wizard) {
```

Comment:
This line is here because we want the main container to stretch by default if there's a wizard or if it has the fill modifier, but not otherwise.